minimint/Cargo.toml

```diff
@@ -9,6 +9,18 @@ edition = "2018"
 [features]
 telemetry = ["opentelemetry", "opentelemetry-jaeger", "tracing-opentelemetry"]
 
+[lib]
+name = "minimint"
+path = "src/lib.rs"
+
+[[bin]]
+name = "minimint"
+path = "src/bin/server.rs"
```

Comment:
Would it be a good idea to rename this file to `minimint.rs` for clarity sake?

Reply:
Let's do in another PR. There's more changes than I'd like here anyway, and I bet something unexpected is going to need another adjustments then. :)